src/tests/evm_tests/evm.test.ts

```diff
@@ -24,7 +24,7 @@ describe('test evm context', () => {
             1
         )
         expect(hash).toBe(
```

Comment:
### At present, we can turn off the hash check first, and turn it on later

expect(hash).not.toBeNull()